### DISCUSSION TITLE
Mapping from All-Atom to CG-SIRAH

### DISCUSSION THREAD

**Discussion by shvel** [SIRAHFF/documentation#85] [Q&A] [answered]
Dear All,

I am trying to better understand the mapping from AA to CG in the SIRAH model and would greatly appreciate your insights on the following points:

Regarding the glutamic acid mapping, how many carbon atoms are included in the BCD (C4Ce) bead?

In the figure depicting the CG lipid representation (available [here](https://sirahff.github.io/documentation/Background%20SIRAH.html)), the A1C bead appears to represent a smaller volume compared to the Y2C beads. If I understand correctly, the Y2C bead contains three carbon atoms (i.e., (CH₂)₃). Could you please clarify how many atoms are represented in the A1C bead?

Thank you very much for your assistance. I greatly appreciate your help!

Best regards,
S. H. V.

**Comment by SIRAHFF** [comment, marked as answer]
Hi SHV,
Actually, our coarse-graining philosophy is not based on the number of atoms being represented by one CG bead. We map the All-Atom to CG models trying to keep intermolecular interaction points. In glutamic acid, for instance, we consider the carboxylate group as the most important part of the side chain and is the part we kept in the CG. Indeed, aspartic acid is very similar to glutamic but with a different distance from the backbone. 

Regarding the lipid part of the question, the A1C bead is indeed smaller because it represents an SP2 carbon and they should be able to interact at closer distances. Again, the size is not related to how many atoms it represents, but the kind of interactions it establishes. For instance, the side chains of Phe are type A1C because they must be able to represent Pi-Pi stacking interactions, while the side chains of hydrophobic (not aromatic) aminoacids are bigger, like Y2C.

I hope this helps, but if not feel free to ask.

Best,
Sergio
